src/components/buttonnavigation/ButtonNavigation.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+import { colors } from "../colors/Colors";
+import { ButtonNavigationCSS } from './ButtonNavigationCSS';
+
+const ButtonNavigation = (props) => {
+    const getVariantStyles = (p) => {
+      const inactiveText = `color: ${colors.grey600}; pointer-events: none`;
```

Comment:
Since this component doesn't have variants, it's fine not to have a switch statement. You can just rename this to something like `getActiveStyles`, remove the switch, and write 
`if (p.active) {
      styles += ${ButtonNavigationCSS};
    } else {
      styles += inactiveText;
    }`